db/migrate/20240207021033_add_limit_to_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddLimitToAccounts < ActiveRecord::Migration[7.2]
+  def change
+    add_column :accounts, :limit, :bigint, default: 0
```

Comment:
Is this field meant to store money? If so, should use the `limit_cents` convention, as per a [previous discussion](https://github.com/maybe-finance/maybe/pull/268)?

Reply:
I was not sure that limit was also supposed to be money field. @Shpigford Can you confirm if we want to monetize limit field as well which will generate limit_cents and limit_currency in accounts table.

Reply:
I've noticed that #268 introduced `balance_currency` (associated with `balance_cents`. Is there a case where there is a need for separate currency fields for each money fields? Like the balance be in USD, but the limit in CAD for example?

Is there a way to have just one currency field, that can be shared by all money fields?

Reply:
Yeah, even I was thinking on same. Even there is separate 'currency' field already present in accounts table. Not sure if we still need it as balance_currency is also added in table.